ksql-engine/src/main/java/io/confluent/ksql/topic/TopicCreateInjector.java

```diff
@@ -53,18 +63,25 @@
 
   private final KafkaTopicClient topicClient;
   private final MetaStore metaStore;
+  private final SchemaRegistryClient srClient;
```

Comment:
I'd personally handle this as a separate injector.  Keep this injector focused on topic creation.

I can see that topic creation is _related_ to schema creation - but the injectors are chained anyway.

Reply:
At first I had tried doing that and now I can't remember why I didn't end up going that route. I'll go ahead and do that.